apps/nextjs-website/src/components/organisms/WebinarQuestionsTemplate/WebinarQuestionsTemplate.tsx

```diff
@@ -0,0 +1,118 @@
+'use client';
+import SummaryInformation from '@/components/atoms/SummaryInformation/SummaryInformation';
+import {
+  Paper,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+} from '@mui/material';
+import { Webinar } from '@/lib/types/webinar';
+import { WebinarQuestion } from '@/lib/webinars/webinarQuestions';
+import { useWebinar } from '@/helpers/webinar.helpers';
+import { useEffect, useState } from 'react';
+import { getWebinarQuestionList } from '@/lib/webinarApi';
+import { useRouter } from 'next/navigation';
+import { useFormatter } from 'next-intl';
+import { CopyToClipboardButton } from '@pagopa/mui-italia';
+import DOMPurify from 'isomorphic-dompurify';
+import Spinner from '@/components/atoms/Spinner/Spinner';
+
+type WebinarQuestionsTemplateProps = {
+  webinar: Webinar;
+};
+
+const WebinarQuestionsTemplate = ({
+  webinar,
+}: WebinarQuestionsTemplateProps) => {
+  const router = useRouter();
+  const formatter = useFormatter();
+  const [loading, setLoading] = useState<boolean>(true);
+  const [questions, setQuestions] = useState<WebinarQuestion[]>([]);
+  const { webinarState, setWebinar } = useWebinar();
+
+  const fetchQuestions = async () => {
+    const data = await getWebinarQuestionList(webinar.slug).catch(() => {
+      router.replace('/404');
+      return [];
+    });
+
+    const sortedQuestions = [...data].sort(
+      (a: WebinarQuestion, b: WebinarQuestion) =>
+        new Date(b.createdAt).getTime() - new Date(a.createdAt).getTime()
+    );
+    setQuestions(sortedQuestions);
+    setLoading(false);
```

Comment:
There's a problem with users without authorization. Maybe you should setQuestions and setLoading only when getWebinarQuestionList responds